oidc-handler/core/pom.xml

```diff
@@ -0,0 +1,283 @@
+
+                <version>2.5.3</version>
+
+                <inherited>true</inherited>
+
+            </plugin>
+
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>2.6</version>
+                <configuration>
+                    <archive>
+                        <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
+                    </archive>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.7.0</version>
+                <configuration>
+                    <source>8</source>
+                    <target>8</target>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.sling</groupId>
+                <artifactId>maven-sling-plugin</artifactId>
+                <version>2.3.4</version>
+                <configuration>
+                    <slingUrl>http://localhost:8080/system/console</slingUrl>
+                    <user>admin</user>
+                    <password>admin</password>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+    <dependencies>
+
+        <dependency>
+
+            <groupId>org.apache.felix</groupId>
+
+            <artifactId>org.apache.felix.scr</artifactId>
+
+            <version>2.0.10</version>
+
+            <scope>provided</scope>
+
+        </dependency>
+
+
+
+        <dependency>
+
+            <groupId>org.apache.felix</groupId>
+
+            <artifactId>org.apache.felix.scr.annotations</artifactId>
+
+            <version>1.9.6</version>
+
+        </dependency>
+
+        <!-- https://mvnrepository.com/artifact/com.day.cq/cq-content-sync -->
+        <!--<dependency>-->
+            <!--<groupId>com.day.cq</groupId>-->
+            <!--<artifactId>cq-content-sync</artifactId>-->
+            <!--<version>5.6.4</version>-->
+            <!--<scope>provided</scope>-->
+        <!--</dependency>-->
+
+        <dependency>
+            <groupId>org.apache.sling</groupId>
+            <artifactId>org.apache.sling.engine</artifactId>
+            <version>2.4.6</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+
+            <groupId>javax.servlet</groupId>
+            <artifactId>servlet-api</artifactId>
+            <version>2.5</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.osgi</groupId>
+            <artifactId>org.osgi.compendium</artifactId>
+            <version>4.2.0</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.osgi</groupId>
```

Comment:
Please see the sling parent pom for a more recent version of this artifact. IIRC the artifactId has also changed.